pelican/tools/pelican_import.py

```diff
@@ -468,10 +468,9 @@ def get_tumblr_posts(api_key, blogname, offset=0):
                     fmtstr = '<img alt="%s" src="%s" />'
                 content = ''
                 for photo in post.get('photos'):
-                    content += '\n'.join(
-                        fmtstr % (photo.get('caption'),
-                                  photo.get('original_size').get('url')))
-                content += '\n\n' + post.get('caption')
+                    content += '\n'.join(fmtstr % (photo.get('caption'),
+                                                   photo.get('original_size').get('url'))
+                                        for photo in post.get('photos'))
```

Comment:
Is this right? It looks like you're iterating across post.get('photos') on line 473, but it's already doing that on line 470.